tests/queries/0_stateless/03623_exclude_materialize_skip_indexes_on_insert.sql

```diff
@@ -0,0 +1,128 @@
+-- Provide list of secondary indexes to skip on insert
+SET use_skip_indexes_on_data_read = 0;
+SET use_query_condition_cache = 0;
```

Comment:
`SET use_query_condition_cache = 0;` isn't necessary for this test --> can be removed.